### ISSUE TITLE
Standardizing and modernizing CMPLXFOIL

### ISSUE THREAD

**Issue by bernardopacini** [mdolab/CMPLXFOIL#18]
# Description of feature

@A-CGray, @eytanadler, and I met with with @whophil and Co to discuss how to recombine two divergent forks of CMPLXFOIL (this one and their internal one). The main points from the meeting are:

- Both teams are using CMPLXFOIL for quick airfoil analysis
- The MDO Lab application is mostly for gradient-based optimization, standardized within the MACH framework (and potentially MPHYS in the long run)
- @whophil and Co are not using the gradient-based side of CMPLXFOIL and do not want the overhead dependencies of the MACH framework
- There is interest from both teams to have a centralized XFOIL library that can be imported into Python, for other projects to build off of.

The main issue in merging the two branches is the different applications, however this can be handled as Python-only differences using the same underlying Fortran code. With this in mind, we propose the following solution and areas to work on (@whophil and others please edit these as needed).

# Potential solution
- [ ] Remove `prefoil` as a dependency so that CMPLXFOIL as a repository only depends on
- [ ] Package CMPLXFOIL into a Conda package
- [ ] Establish that both teams' Python wrappers still work with the same underlying source code.
- [ ] (Stretch goal) Add tests for the Fortran layer of the code.

@eytanadler, @A-CGray, @whophil, let me know if this sounds reasonable, we can keep the discussion going on this issue page.

**Comment by bernardopacini** [comment]
Thank you @whophil for adding CMPLXFOIL to Conda ([PR linked here](https://github.com/conda-forge/staged-recipes/pull/20684)). Please let us know if you run into any issues with the new workflow, otherwise we can check off our third to-do item here.

**Comment by A-CGray** [comment]
Can we mark this as completed now @bernardopacini ?

**Comment by bernardopacini** [comment]
> Can we mark this as completed now @bernardopacini ?

Yes, that sounds good to me. We don't have extensive tests but given that no one on our side is working with CMPLXFOIL at the moment I think we can tag that as an issue for when someone does pick it up later on.
